Client/Frontend/Browser/Tabs/TabTrayViewModel.swift

```diff
@@ -7,40 +7,75 @@ import Storage
 
 class TabTrayViewModel {
 
+    enum Segment: Int, CaseIterable {
+        case tabs
+        case privateTabs
+        case syncedTabs
+
+        var navTitle: String {
+            switch self {
+            case .tabs:
+                return .TabTrayV2Title
+            case .privateTabs:
+                return .TabTrayPrivateBrowsingTitle
+            case .syncedTabs:
+                return .AppMenu.AppMenuSyncedTabsTitleString
+            }
+        }
+
+        var label: String {
+            switch self {
+            case .tabs:
+                return String.TabTraySegmentedControlTitlesTabs
+            case .privateTabs:
+                return String.TabTraySegmentedControlTitlesPrivateTabs
+            case .syncedTabs:
+                return String.TabTraySegmentedControlTitlesSyncedTabs
+            }
+        }
+
+        var imageName: String {
+            switch self {
+            case .tabs:
+                return ImageIdentifiers.navTabCounter
+            case .privateTabs:
+                return ImageIdentifiers.privateMaskSmall
+            case .syncedTabs:
+                return ImageIdentifiers.syncedDevicesIcon
+            }
+        }
+    }
```

Comment:
Wondering if getting the image instead of just the name would make things more cleaner in other areas. @thatswinnie  🤔